src/main/java/rest/InvoiceResource.kt

```diff
@@ -66,4 +63,11 @@ class InvoiceResource @Inject constructor(
     @Produces("application/json")
     fun allInvoicesByState(@PathParam("state") state: String): List<Invoice> =
             invoiceService.allInvoicesByState(InvoiceState.valueOf(state))
+
+    @POST
+    @Path("/state/change/")
+    @Produces("application/json")
+    fun updateInvoiceState(@FormParam("invoiceId") invoiceId: String,
```

Comment:
Has this endpoint been properly tested?

Reply:
Added @Consumes for now, later this will be changed. Created new JIRA [PTS6-113] Issue to do this.